### PR TITLE
fix: nft assets ui

### PR DESCRIPTION
**Pull request by zhouhanseng**

### CODE REVIEW

File: packages/maskbook/src/plugins/Wallet/apis/opensea.ts [hunk 1]
```diff
@@ -123,7 +123,6 @@ export async function getAssetsList(from: string, size = 50, page = 0) {
     const response = await fetch(`https://api.opensea.io/api/v1/assets?${params.toString()}`, {
         method: 'GET',
         mode: 'cors',
-        cache: 'force-cache',
```
Comment (zhouhanseng) [inline packages/maskbook/src/plugins/Wallet/apis/opensea.ts:126]: I owned my new first asset. But with `cache: 'force-cache'`, it still loads empty array.

File: packages/maskbook/src/extension/options-page/DashboardComponents/CollectibleList/Card.tsx [hunk 2]
```diff
@@ -40,7 +40,7 @@ export function CollectibleCard(props: CollectibleCardProps) {
                         height: 220,
                     }}>
                     {props.url ? (
-                        <Image component="img" width={160} height={220} src={props.url} />
+                        <Image component="img" style={{ width: '90%', height: 'auto' }} src={props.url} />
```
Comment (zhouhanseng) [inline packages/maskbook/src/extension/options-page/DashboardComponents/CollectibleList/Card.tsx:43]: fixed width and height would stretch the image inappropriately 




to

**Review by Jack-Works — DISMISSED** [review]

File: packages/maskbook/src/extension/options-page/DashboardComponents/CollectibleList/Card.tsx (hunk 2 above)
Comment (guanbinrui) [reply packages/maskbook/src/extension/options-page/DashboardComponents/CollectibleList/Card.tsx:43]: The `object-fit` rule is invented for this. Have it a try.

Comment (zhouhanseng) [reply packages/maskbook/src/extension/options-page/DashboardComponents/CollectibleList/Card.tsx:43]: It works, and is more readable! thanks

**Review by guanbinrui — APPROVED** [review]

**Pull request merged**